### ISSUE TITLE
semantic highlighting of cached_property

### ISSUE THREAD

**Issue by amotzop** [microsoft/pylance-release#3742]
## Environment data

-   Language Server version: 2022.12.21
-   OS and version: ubuntu  20.04.4
-   Python version (& distribution if applicable, e.g. Anaconda): python3.9

## Code Snippet
```python
from functools import cached_property

class Cls:
    @property
    def property(self):
        ...

    @cached_property
    def cached_property(self):
        ...
```

## Expected behavior
I think both the `property` and the `cached_property` should have the semantic highlighting of a property
(In the hover provider they are both categorized as `(property) ...`

## Actual behavior
The `cached_propery` has the semantic highlighting of a method.

## Remark
This was already noted in #489, but that issue was closed (for a different reason from my understanding, it brunched into another issue which was closed).

**Comment by rchiodo** [comment]
![image](https://user-images.githubusercontent.com/19672699/206754344-e4f1821d-dce5-4479-96d5-5687bc4aec12.png)

My guess is we special case `property` in the semantic tokenizer.

**Comment by bschnurr** [comment]
This issue has been fixed in prerelease version 2023.2.11, which we've just released. You can find the changelog here: [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#2023211-1-february-2023-prerelease)
